File: Giovanna-e-Gabriela/CPP/Lista-1/ex-05.cpp
/*
Data: 05/05/2026
Autores: Gabriela Onaga e Giovanna Costa 
Descrição: Ler um número para a parada final e outro número que 
representa um múltiplo. Exibir os múltiplos do número lido de 1 
até o número final lido.
/*

#include <iostream>
using namespace std;

int main() {
    int limite, base;

    cout << "Digite o numero final: ";
    cin >> limite;

    cout << "Digite o numero base: ";
    cin >> base;

    for (int i = 1; i <= limite; i++) {
        if (i % base == 0) {
            cout << i << " ";
        }
    }

    return 0;
}

File: Giovanna-e-Gabriela/Python/python-conversion/python/ex05.py
"""
    Autor: Giovanna Cabral e Gabriela Onaga
    Data: 12/08/2026
    Descrição: Lê um número de 1 a 10 e exibe sua tabuada, validando a entrada.
"""

num = int(input("Digite um numero de 1 a 10: "))
contador = 1

while num < 1 or num > 10:
    num = int(input("Digite um numero de 1 a 10: "))

while contador <= 10:
    print(f"{num} x {contador} = {num * contador}")
    contador += 1
